sqlnexus/resources/CreateDropDB.txt

```diff
@@ -4,7 +4,7 @@ begin
 	alter database [{0}] set single_user with rollback immediate
 	drop database [{0}];
 end
-create database [{0}];
+create database [{0}]  collate Latin1_General_CI_AS;
```

Comment:
Based on code review of current code base, COLLATE in this file has been removed altogether. Therefore the database will inherit what SQL Server is installed with by default (Model db)
Thank you